runtime/pack/dist/opt/termdebug/plugin/termdebug.vim

```diff
@@ -802,12 +802,13 @@ func s:InstallCommands()
   command Finish call s:SendCommand('-exec-finish')
   command -nargs=* Run call s:Run(<q-args>)
   command -nargs=* Arguments call s:SendCommand('-exec-arguments ' . <q-args>)
-  command Stop call s:SendCommand('-exec-interrupt')
 
   " using -exec-continue results in CTRL-C in gdb window not working
   if s:way == 'prompt'
+    command Stop call s:PromptInterrupt()
     command Continue call s:SendCommand('continue')
   else
+    command Stop call s:SendCommand('-exec-interrupt')
     command Continue call term_sendkeys(s:gdbbuf, "continue\r")
```

Comment:
@k-takata Do you have any idea why `gdbbuf` is used here? I _think_ it should be `combuf? in which case we can just use `SendCommand`...